scrapers/AVE.yml

```diff
@@ -0,0 +1,68 @@
+name: AVEntertainments
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - aventertainments.com
+    scraper: sceneScraper
+
+#search by id is very picky, make sure to search using the full id eg XXX-014
+
+#choose one of the below URLs and replace in the 'queryURL:' line
+#search JAV DVD
+#https://www.aventertainments.com/search_Products.aspx?languageID=1&dept_id=29&searchby=keyword&keyword={}
+#search JAV PPV
+#https://www.aventertainments.com/ppv/ppv_searchproducts.aspx?languageID=1&vodtypeid=1&keyword={}
+#search Western DVD
+#https://www.aventertainments.com/search_Products.aspx?languageID=1&dept_id=43&searchby=keyword&keyword={}}
+#search Western PPV
+#https://www.aventertainments.com/ppv/ppv_searchproducts.aspx?languageID=1&vodtypeid=2&keyword={}
+
+sceneByName:
+  action: scrapeXPath
+  queryURL: https://www.aventertainments.com/search_Products.aspx?languageID=1&dept_id=29&searchby=keyword&keyword={}
+  scraper: sceneSearch
+sceneByQueryFragment:
+  action: scrapeXPath
+  queryURL: "{url}"
+  scraper: sceneScraper
+
+xPathScrapers:
+  sceneSearch:
+    common:
+      $searchRes: //div[@class="single-slider-product grid-view-product"]
+      $searchTitle: //div[@class="single-slider-product grid-view-product"]//p[@class="product-title"]
+    scene:
+      Title: $searchTitle/a
+      URL: $searchTitle/a/@href
+      Image: $searchRes/div[@class="single-slider-product__image"]/a/img/@src
+  sceneScraper:
+    common:
+      $singleInfo: //div[@class="single-info"]
+      $performer: //div[@class="single-info"][contains(span,"Starring")]/span[@class="value"]/a
+    scene:
+      Title:
+        selector: $singleInfo[contains(span,"Item#")]/span[@class="tag-title"]
```

Comment:
I think `Item#` should be just `Item`, found a scene with a space so the selector give nothing.

https://www.aventertainments.com/ppv/new_detail.aspx?ProID=13757&languageID=1&VODTypeID=1

Reply:
Nice catch. Yeah didnt notice that. I updated the selector using starts-with and plain `Item`